common/tools/dev-tool/src/util/prompt.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import readline from "node:readline";
```

Comment:
there's a promise version of this module too!

`import * as readline from 'node:readline/promises';`